casper/src/main/resources/MakeMint.rho

```diff
@@ -63,10 +82,33 @@ in {
               new result in {
                 @src!(bundle0{bundle0{src}|*decr}, amount, *result) | //call src decr function.
                 for(@decrSuccess, @auth <- result) {
-                  if (decrSuccess and auth == *decr) {
-                    balance!("add", amount, *success) // add transferred amount to this purse
-                  } else {
-                    success!(false) 
+                  for (logCh <<- logStore) {
+                    if (decrSuccess and auth == *decr) {
+                      // add transferred amount to this purse
+                      balance!("add", amount, *success) |
+                      
+                      // log
+                      if (Nil != *logCh) {
+                        new bd(`rho:block:data`), bdCh in {
+                          bd!(*bdCh) |
+                          for (@timestamp, @blockNumber, @sender <- bdCh) {
+                            logCh!(["deposit", amount, src.toByteArray(), true, timestamp, blockNumber, sender])
+                          }
+                        }
+                      }
+                    } else {
+                      success!(false) |
+                      
+                      // log
+                      if (Nil != *logCh) {
+                        new bd(`rho:block:data`), bdCh in {
+                          bd!(*bdCh) |
+                          for (@timestamp, @blockNumber, @sender <- bdCh) {
```

Comment:
Block number is on the first place - [SystemProcesses.scala](https://github.com/rchain/rchain/blob/f6050234/rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala#L263).

Reply:
Thanks.  I've fixed it everywhere in my tests and in BlockDataContractTest.rho where I copied it from.